src/pages/settings/Security/TwoFactorAuth/StepWrapper/StepWrapper.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from "prop-types";
+import HeaderWithBackButton from '../../../../../components/HeaderWithBackButton';
+import ScreenWrapper from '../../../../../components/ScreenWrapper';
+import FullPageOfflineBlockingView from '../../../../../components/BlockingViews/FullPageOfflineBlockingView';
+import * as TwoFactorAuthActions from '../../../../../libs/actions/TwoFactorAuthActions';
+
+const propTypes = {
+    /** Title of the Header */
+    title: PropTypes.string,
+
+    /** Data to display a step counter in the header */
+    stepCounter: PropTypes.shape({
+        step: PropTypes.number,
+        total: PropTypes.number,
+        text: PropTypes.string,
+    }),
+
+    /** Method to trigger when pressing back button of the header */
+    onBackButtonPress: PropTypes.func,
+
+    /** Children components */
+    children: PropTypes.node,
+}
+
+const defaultProps = {
```

Comment:
Use prop destructuring with default values instead of `defaultProps` -> https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring